app/src/main/java/com/example/mycalculator/StringUtils.java

```diff
@@ -0,0 +1,6 @@
+package com.example.mycalculator;
```

Comment:
yo can create a package called utils, and put this class inside

Reply:
Done. Thx!